docker/sane-pdf-reports/Dockerfile

```diff
@@ -33,11 +31,17 @@ RUN apt-get update -y -q && \
 
 RUN npm i puppeteer@5.3.1 && ln -s /app/node_modules/puppeteer/.local-chromium/linux-*/chrome-linux/chrome /usr/bin/chromium-browser
 
-RUN python get_build_artifacts.py --project sane-reports --branch master --filter "sane-report$" \
-    && mv sane-report sane-report.tar && tar -xvf sane-report.tar && rm -f sane-report.tar
+RUN apt-get update -y -q && apt-get install -y -q apt-transport-https ca-certificates gnupg curl \
+    && echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list \
+    && curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | gpg --dearmor -o /usr/share/keyrings/cloud.google.gpg && apt-get update -y && apt-get install google-cloud-sdk -y
```

Comment:
Can you write in words what do you do here?
Maybe we should add comments explaining this?

Reply:
adding the sources where google cloud can be installed from. I just followed these docs, i'll add it in a comment here
https://cloud.google.com/storage/docs/gsutil_install